add `soil_conductivity` slot descriptions

mslarae13 opened the pull request:
https://github.com/GenomicsStandardsConsortium/mixs/issues/813 

Add descriptions for these slots:

- warning Slot 'soil_conductivity' does not have recommended slot 'description' (recommended)

mslarae13 commented on src/mixs/schema/mixs.yaml:
```diff
@@ -12287,6 +12287,7 @@ slots:
       interpolated: true
       partial_match: true
   soil_conductivity:
+    description: Soil electrical conductivity (EC) is a measure of how well soil water can carry an electrical current. Electrical conductivity of soil solution is a reliable indicator of its solute (cation or anion) concentration with 1 dS/m approximately equivalent to 10 meq/L (U.S. Salinity Laboratory Staff, 1954).
```
@only1chunts @lschriml 
This slot was added by "The AgMicrobiome project, JP Dundore-Arias, Linda Kinkel"
Do you have a way to contact someone to check in that this description fits their goal for this slot?

lschriml commented:
Hello Montana,
 I can contact them.
I worked on MIxS-Ag.

Question: where did this new definition come from ?

Looking at MIxS 6.0:  the definition of this term was: conductivity of some
soil.

As I mentioned in a previous email this morning, once a package has been
approved,
we generally do not rewrite the content, e.g., definitions. Other than
formatting/grammar.

Agreed, that if we identify a change that is needed it is the correct step
to contact those that created the checklist/extension.

Cheers,
Lynn


On Wed, Jul 17, 2024 at 7:06 PM Montana ***@***.***> wrote:

> ***@***.**** commented on this pull request.
> ------------------------------
>
> In src/mixs/schema/mixs.yaml
> <https://github.com/GenomicsStandardsConsortium/mixs/pull/826#discussion_r1681869583>
> :
>
> > @@ -12287,6 +12287,7 @@ slots:
>        interpolated: true
>        partial_match: true
>    soil_conductivity:
> +    description: Soil electrical conductivity (EC) is a measure of how well soil water can carry an electrical current. Electrical conductivity of soil solution is a reliable indicator of its solute (cation or anion) concentration with 1 dS/m approximately equivalent to 10 meq/L (U.S. Salinity Laboratory Staff, 1954).
>
> @only1chunts <https://github.com/only1chunts> @lschriml
> <https://github.com/lschriml>
> This slot was added by "The AgMicrobiome project, JP Dundore-Arias, Linda
> Kinkel"
> Do you have a way to contact someone to check in that this description
> fits their goal for this slot?
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/GenomicsStandardsConsortium/mixs/pull/826#pullrequestreview-2184239742>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ABBB4DJTO6AQ5H4FMGS6VGDZM32FZAVCNFSM6AAAAABLBQ2WE6VHI2DSMVQWIX3LMV43YUDVNRWFEZLROVSXG5CSMV3GSZLXHMZDCOBUGIZTSNZUGI>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>


-- 
Lynn M. Schriml, Ph.D.
Associate Professor

Institute for Genome Sciences
University of Maryland School of Medicine
Department of Epidemiology and Public Health
670 W. Baltimore St., HSFIII, Room 3061
Baltimore, MD 21201
P: 410-706-6776 |  F: 410-706-6756
***@***.***

mslarae13 commented:
@lschriml 

If the previous definition was "conductivity of some soil" that somehow got lost as there's no description now!
We can make the update to just include the original definition, however, I would argue and have a strong opinion against definitions that re-use the word. That's not a definition/description and isn't helpful.

conductivity is the conductivity means nothing to someone who is looking for a description.

I found this description by looking into what is soil conductivity. It might not be right! And that's why I want to confirm with those that created the extension. But I do think we should provide a better and more informative description.

lschriml commented:
A good solution.Let’s use their first definition, as it was in version 6 of MIxS. Let me know if you do not have that spreadsheet. In the past we kept the latest release in GitHub, so that we could check back on what was done before.We can definitely send a request to the submitters to ask if the updated definition aligns for them to what they intended.   We can put together a contact do, for who to contact for each standard. I documented on the website, for each checklist/extension, who created it.Genomic Standards Consortiumgensc.orgCheers,LynnSent from my iPhoneOn Jul 19, 2024, at 6:21 PM, Montana ***@***.***> wrote:﻿
@lschriml
If the previous definition was "conductivity of some soil" that somehow got lost as there's no description now!
We can make the update to just include the original definition, however, I would argue and have a strong opinion against definitions that re-use the word. That's not a definition/description and isn't helpful.
conductivity is the conductivity means nothing to someone who is looking for a description.
I found this description by looking into what is soil conductivity. It might not be right! And that's why I want to confirm with those that created the extension. But I do think we should provide a better and more informative description.

—Reply to this email directly, view it on GitHub, or unsubscribe.You are receiving this because you were mentioned.Message ID: ***@***.***>

lschriml commented:
Other than grammatical changes, we do not commonly edit term definitions after a standard was put together by the domain experts.For the past two releases we did do a deeper dive to standardize terms, definition formatting.This is a good opportunity, to document our update process, what we do and do not change after a standard is established and released.And to clarify what the CIG does and doesn’t edit between releases. And the steps to suggest a content update.This is also a point for us to think about what types of work we are doing each month, what are our goals for the next year and charting a path forward.I would say that getting the new standards reviewed, finalizing and documenting the editing and release process in place -  be a primary goal. Review of allowable values, term definitions is important work, but it is good to consider that these things were thought about long and hard when they were developed. Granted the style of each sedition is not perfect, nor do they all conform to an exact model. They have been developed over nearly 20 years, so that is to be expected. They do not have to be perfect or entirely uniform.      I want us to maximize our efforts to further our mission, to build and release new standards.  To accomplish our plan to provide MIxS in a standardized format. And support the communities building those standards.Cheers,LynnSent from my iPhoneOn Jul 19, 2024, at 7:44 PM, Lynn Schriml ***@***.***> wrote:﻿A good solution.Let’s use their first definition, as it was in version 6 of MIxS. Let me know if you do not have that spreadsheet. In the past we kept the latest release in GitHub, so that we could check back on what was done before.We can definitely send a request to the submitters to ask if the updated definition aligns for them to what they intended.   We can put together a contact do, for who to contact for each standard. I documented on the website, for each checklist/extension, who created it.Genomic Standards Consortiumgensc.org<favicon.ico>Cheers,LynnSent from my iPhoneOn Jul 19, 2024, at 6:21 PM, Montana ***@***.***> wrote:﻿
@lschriml
If the previous definition was "conductivity of some soil" that somehow got lost as there's no description now!
We can make the update to just include the original definition, however, I would argue and have a strong opinion against definitions that re-use the word. That's not a definition/description and isn't helpful.
conductivity is the conductivity means nothing to someone who is looking for a description.
I found this description by looking into what is soil conductivity. It might not be right! And that's why I want to confirm with those that created the extension. But I do think we should provide a better and more informative description.

—Reply to this email directly, view it on GitHub, or unsubscribe.You are receiving this because you were mentioned.Message ID: ***@***.***>

mslarae13 commented:
@lschriml 

I changed the description back to the original description.
I moved my clarification about what soil conductivity really is to a comment. I have a strong opinion about definitions that just re-use the word they're defining! It's bothers me 😅

Is this sufficient? So I brought back the original description that got lost.

lschriml reviewed (approved):
reviewed and approve changes.
Lynn
